### DISCUSSION TITLE
"gateway timeout" in Grafana

### DISCUSSION THREAD

**Discussion by LuxorN09** [tillsteinbach/VWsFriend#713] [Q&A]
I have a little question. 
My car is displayed correct in the first tile in Grafana, the status is correct also.
Clicking on it takes me to VWsFriend (Port 4000), everything is fine.

However, next to the name of my vehicle is and an exclamation mark in a red triangle with the mouseover text "gateway timeout".
How do I get rid of this?
If this is important, I don't use HomeKit integration.

The error is below:
`Gateway Timeout
Object
status:504
statusText:"Gateway Timeout"
data:Object
message:"Gateway Timeout"
error:"Gateway Timeout"
response:""
config:Object
url:"api/datasources/proxy/uid/P36D08C1A0297C737/status/vehicles/WVWZZZEDXSE009324-status_or_car-badge.png.json"
method:"GET"
headers:Object
retry:0
hideFromInspector:false
traceId:undefined
message:"Gateway Timeout"`

Thank you in advance.

**Comment by tillsteinbach** [comment]
do you have a screenshot?

**Reply by LuxorN09** [reply]
> do you have a screenshot?

![grafana](https://github.com/user-attachments/assets/b5627ade-c695-4c11-9101-22836540462f)
